molecule/verify-workers.yml

```diff
@@ -0,0 +1,32 @@
+---
+
+- name: Debug workers node
+  delegate_to: "{{ groups['control_plane'][0] }}"
+  block:
+    - name: Query health endpoint to show output
+      ansible.builtin.shell: |
+        kubectl get node {{ ansible_hostname }}
+      register: kubectl_show
+      changed_when: false
+
+    - name: Debug kubectl_show for {{ ansible_hostname }}
+      ansible.builtin.debug:
+        var: kubectl_show
+
+- name: Check that workers nodes are ready
+  delegate_to: "{{ groups['control_plane'][0] }}"
+  block:
+    - name: Check for node to be ready
+      ansible.builtin.pause:
+        minutes: 3
```

Comment:
Think about changing to the following scheme: checking `http://localhost:10248/healthz` endpoint until 200 status

Note: check please what this url actually shows during worker node initialization